src/types/ListTypes.ts

```diff
@@ -17,6 +17,7 @@ export type RegistrationData = {
     email: string;
     name: string;
     surname: string;
+    phone?: string | null,
```

Comment:
nie jestem pewien, to nam daje typ: 
phone: string | null | undefined
mozesz juz zostawic ale chyba samo phone?: string by wystarczylo :)